templates/localeFile.tpl

```diff
@@ -6,77 +6,169 @@
  *
  *}
 
-<script type="text/javascript">
-	$(function() {ldelim}
-		// Attach the form handler.
-		$('#localeFilesForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
-	{rdelim});
-</script>
+<form class="pkp_form" id="localeFilesForm" method="post" action="{url router=$smarty.const.ROUTE_COMPONENT component="plugins.generic.customLocale.controllers.grid.CustomLocaleGridHandler" op="updateLocale" locale=$locale key=$filePath anchor="localeContents"}">
+	<link rel="stylesheet" href="{$baseUrl}/plugins/generic/customLocale/css/customLocale.css" type="text/css" />
+	<div id="customLocales">
+		{* TABLE *}
+		<table class="pkpTable">
+			{* TABLE HEADER *}
+			<caption>
+				<span class="pkpHeader__title">
+					<h3>{translate key="plugins.generic.customLocale.file.editHeader"}</h3>
+				</span>
+				<div class="pkpHeader__actions">
+					{* SEARCH BOX *}
+					<div class="pkpSearch">
+						<label>
+							<span class="-screenReader">{translate key="common.search"}</span>
+							<input
+								type="search"
+								id="1351"
```

Comment:
Just curious -- where did this magic number come from? If it's just for the sake of the `aria-controls` attribute on the `<button>`, it would be better to use something symbolic, I think.

Reply:
Yeah I'd recommend something namespaced with the plugin's name like `customLocale__searchInput`.

Reply:
It was for the `aria-controls`. I changed it to `customLocale__searchInput`.